entries/p/portal.arubainstanton.com.json

```
@@ -0,0 +1,12 @@
+{
+  "ArubaInstantOn": {
+    "domain": "portal.arubainstanton.com",
```

```suggestion
    "domain": "arubainstanton.com",
```

Even though the community and the portal use separate authentication systems, the base domain should be listed.

---

So, the only reference to the subdomain should be in the name of this json file?